lib/dirac.js

```diff
@@ -325,7 +325,9 @@ dirac.createViews = function( callback ){
         dirac.raw( query, done );
       };
     })
-  , callback
+  , function( error ){
+      return callback( error );
```

Comment:
Ensure that series doesn't return any results to folks listening. They really mean nothing